Mobile Test Runner for Jama/Mobile Test Runner for Jama/ProjectCollectionViewCell.swift

```diff
@@ -10,4 +10,19 @@ import UIKit
 
 class ProjectCollectionViewCell: UICollectionViewCell {
     @IBOutlet weak var projectCellLabel: UILabel!
+    
+    override func touchesBegan(_ touches: Set<UITouch>, with event: UIEvent?) {
+        super.touchesBegan(touches, with: event)
+        self.backgroundColor = UIColor(colorLiteralRed: 0xF5/0xFF, green: 0xF5/0xFF, blue: 0xF5/0xFF, alpha: 1)
```

Comment:
Are the projects supposed to turn red when selected? In the simulator they become a little grey - is that what this line is doing?